assets/src/pages/room/RoomPage.tsx

```diff
@@ -98,17 +100,34 @@ const RoomPage: FC<Props> = ({
     messageComparator
   );
 
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+
   return (
     <PageLayout>
       <div className="flex h-full w-full flex-col gap-y-4">
         {/* main grid - videos + future chat */}
-        <section className="flex h-full w-full flex-col">
+        <section
+          className={clsx(
+            "flex h-full w-full self-center justify-self-center 3xl:max-w-[1728px]",
+            isSidebarOpen && "gap-x-4"
+          )}
+        >
           <VideochatSection
             peers={peerState.remote}
             localPeer={peerState.local}
             showSimulcast={showSimulcastMenu}
             webrtc={webrtc}
           />
+
+          <div
+            className={clsx(
+              isSidebarOpen ? "max-w-[300px]" : "max-w-0",
```

Comment:
Do we need `max-w[300px]` if the `Sidebar` component itself has fixed height?

Reply:
I think so, we need this to animate the width transition